src/main/java/dbProcs/Getter.java

```diff
@@ -65,96 +76,142 @@
 	 * @return A string array made up of nothing or information to be consumed by the initiating authentication process.
 	 */
 
-	public static String[] authUser (String ApplicationRoot, String userName, String password)
-	{
+	public static String[] authUser(String ApplicationRoot, String userName, String password) {
 		String[] result = null;
 		log.debug("$$$ Getter.authUser $$$");
-		log.debug("userName = "  + userName);
+		
+		log.debug("userName = " + userName);
 
 		boolean userFound = false;
-		boolean goOn = false;
+		boolean userVerified = false;
+
 		Connection conn = Database.getCoreConnection(ApplicationRoot);
-		try
-		{
-			//See if user Exists
-			CallableStatement callstmt = conn.prepareCall("call userFind(?)");
-			log.debug("Gathering userFind ResultSet");
+
+		// See if user Exists
+		CallableStatement callstmt;
+		try {
+			callstmt = conn.prepareCall(
+					"SELECT userId, userName, userPass, userRole, badLoginCount, tempPassword, classId, suspendedUntil FROM `users` WHERE userName = ?");
+		} catch (SQLException e) {
+			log.fatal("Could not retrieve users from database: " + e.toString());
+			throw new RuntimeException(e);
+		}
+
+		log.debug("Gathering userFind ResultSet");
+		ResultSet userFind;
+		try {
 			callstmt.setString(1, userName);
-			ResultSet userFind = callstmt.executeQuery();
-			log.debug("Opening Result Set from userFind");
-			try
-			{
-				userFind.next();
-				log.debug("User Found"); //User found if a row is in the database, this line will not work if the result set is empty
-				userFound = true;
-			}
-			catch(Exception e)
-			{
-				log.debug("User did not exist");
-				userFound = false;
+			userFind = callstmt.executeQuery();
+		} catch (SQLException e) {
+			log.fatal("Could not create call statement: " + e.toString());
+			throw new RuntimeException(e);
+		}
+
+		log.debug("Opening Result Set from userFind");
```

Comment:
If we're removing the "userFind" stored procedure from the authUser message, the logging should be updated to match

Reply:
Will do.